resources/airbnb.swiftformat

```diff
@@ -1,5 +1,5 @@
 # Current version of SwiftFormat used at Airbnb: 
-# https://github.com/calda/SwiftFormat/releases/tag/0.49-beta-1
+# https://github.com/calda/SwiftFormat/releases/tag/0.49-beta-2
```

Comment:
I bumped the SwiftFormat version to 0.49 beta 2, which includes https://github.com/nicklockwood/SwiftFormat/pull/1064